src/state/tool/api/auth.endpoint.jsx

```diff
@@ -0,0 +1,13 @@
+import {API_BASE_URL} from "./index";
```

Comment:
So a lot of these basic configuration files can be JS files, if we do not need to render react components, then we don't need to define a jsx file type we can just define a js file type.